src/rz/angular/effects/service/service.spec.ts

```diff
@@ -23,4 +25,23 @@ describe('exportServiceFile', () => {
         const expected = readFileSync('src/rz/angular/effects/service/snapshots/index-output.ts.snap').toString();
         expect(result).toEqual(expected);
     });
+    it('should trigger standalone component effects', () => {
```

Comment:
We will add a unit test for `filesToAffect` as well

Reply:
filesToAffect would be in morph.spec.ts?